// es-abstract/es2015.js
'use strict';

/* eslint global-require: 0 */
// https://www.ecma-international.org/ecma-262/6.0/#sec-abstract-operations
var ES6 = {
	'Abstract Equality Comparison': require('es-abstract/2015/AbstractEqualityComparison'),
	'Abstract Relational Comparison': require('es-abstract/2015/AbstractRelationalComparison'),
	'Strict Equality Comparison': require('es-abstract/2015/StrictEqualityComparison'),
	AdvanceStringIndex: require('es-abstract/2015/AdvanceStringIndex'),
	ArrayCreate: require('es-abstract/2015/ArrayCreate'),
	ArraySetLength: require('es-abstract/2015/ArraySetLength'),
	ArraySpeciesCreate: require('es-abstract/2015/ArraySpeciesCreate'),
	Call: require('es-abstract/2015/Call'),
	CanonicalNumericIndexString: require('es-abstract/2015/CanonicalNumericIndexString'),
	CompletePropertyDescriptor: require('es-abstract/2015/CompletePropertyDescriptor'),
	CreateDataProperty: require('es-abstract/2015/CreateDataProperty'),
	CreateDataPropertyOrThrow: require('es-abstract/2015/CreateDataPropertyOrThrow'),
	CreateHTML: require('es-abstract/2015/CreateHTML'),
	CreateIterResultObject: require('es-abstract/2015/CreateIterResultObject'),
	CreateListFromArrayLike: require('es-abstract/2015/CreateListFromArrayLike'),
	CreateMethodProperty: require('es-abstract/2015/CreateMethodProperty'),
	DateFromTime: require('es-abstract/2015/DateFromTime'),
	Day: require('es-abstract/2015/Day'),
	DayFromYear: require('es-abstract/2015/DayFromYear'),
	DaysInYear: require('es-abstract/2015/DaysInYear'),
	DayWithinYear: require('es-abstract/2015/DayWithinYear'),
	DefinePropertyOrThrow: require('es-abstract/2015/DefinePropertyOrThrow'),
	DeletePropertyOrThrow: require('es-abstract/2015/DeletePropertyOrThrow'),
	EnumerableOwnNames: require('es-abstract/2015/EnumerableOwnNames'),
	FromPropertyDescriptor: require('es-abstract/2015/FromPropertyDescriptor'),
	Get: require('es-abstract/2015/Get'),
	GetIterator: require('es-abstract/2015/GetIterator'),
	GetMethod: require('es-abstract/2015/GetMethod'),
	GetOwnPropertyKeys: require('es-abstract/2015/GetOwnPropertyKeys'),
	GetPrototypeFromConstructor: require('es-abstract/2015/GetPrototypeFromConstructor'),
	GetSubstitution: require('es-abstract/2015/GetSubstitution'),
	GetV: require('es-abstract/2015/GetV'),
	HasOwnProperty: require('es-abstract/2015/HasOwnProperty'),
	HasProperty: require('es-abstract/2015/HasProperty'),
	HourFromTime: require('es-abstract/2015/HourFromTime'),
	InLeapYear: require('es-abstract/2015/InLeapYear'),
	InstanceofOperator: require('es-abstract/2015/InstanceofOperator'),
	Invoke: require('es-abstract/2015/Invoke'),
	IsAccessorDescriptor: require('es-abstract/2015/IsAccessorDescriptor'),
	IsArray: require('es-abstract/2015/IsArray'),
	IsCallable: require('es-abstract/2015/IsCallable'),
	IsConcatSpreadable: require('es-abstract/2015/IsConcatSpreadable'),
	IsConstructor: require('es-abstract/2015/IsConstructor'),
	IsDataDescriptor: require('es-abstract/2015/IsDataDescriptor'),
	IsExtensible: require('es-abstract/2015/IsExtensible'),
	IsGenericDescriptor: require('es-abstract/2015/IsGenericDescriptor'),
	IsInteger: require('es-abstract/2015/IsInteger'),
	IsPromise: require('es-abstract/2015/IsPromise'),
	IsPropertyDescriptor: require('es-abstract/2015/IsPropertyDescriptor'),
	IsPropertyKey: require('es-abstract/2015/IsPropertyKey'),
	IsRegExp: require('es-abstract/2015/IsRegExp'),
	IteratorClose: require('es-abstract/2015/IteratorClose'),
	IteratorComplete: require('es-abstract/2015/IteratorComplete'),
	IteratorNext: require('es-abstract/2015/IteratorNext'),
	IteratorStep: require('es-abstract/2015/IteratorStep'),
	IteratorValue: require('es-abstract/2015/IteratorValue'),
	MakeDate: require('es-abstract/2015/MakeDate'),
	MakeDay: require('es-abstract/2015/MakeDay'),
	MakeTime: require('es-abstract/2015/MakeTime'),
	MinFromTime: require('es-abstract/2015/MinFromTime'),
	modulo: require('es-abstract/2015/modulo'),
	MonthFromTime: require('es-abstract/2015/MonthFromTime'),
	msFromTime: require('es-abstract/2015/msFromTime'),
	ObjectCreate: require('es-abstract/2015/ObjectCreate'),
	OrdinaryDefineOwnProperty: require('es-abstract/2015/OrdinaryDefineOwnProperty'),
	OrdinaryGetOwnProperty: require('es-abstract/2015/OrdinaryGetOwnProperty'),
	OrdinaryHasInstance: require('es-abstract/2015/OrdinaryHasInstance'),
	OrdinaryHasProperty: require('es-abstract/2015/OrdinaryHasProperty'),
	RegExpExec: require('es-abstract/2015/RegExpExec'),
	RequireObjectCoercible: require('es-abstract/2015/RequireObjectCoercible'),
	SameValue: require('es-abstract/2015/SameValue'),
	SameValueZero: require('es-abstract/2015/SameValueZero'),
	SecFromTime: require('es-abstract/2015/SecFromTime'),
	Set: require('es-abstract/2015/Set'),
	SetFunctionName: require('es-abstract/2015/SetFunctionName'),
	SetIntegrityLevel: require('es-abstract/2015/SetIntegrityLevel'),
	SpeciesConstructor: require('es-abstract/2015/SpeciesConstructor'),
	SymbolDescriptiveString: require('es-abstract/2015/SymbolDescriptiveString'),
	TestIntegrityLevel: require('es-abstract/2015/TestIntegrityLevel'),
	thisBooleanValue: require('es-abstract/2015/thisBooleanValue'),
	thisNumberValue: require('es-abstract/2015/thisNumberValue'),
	thisStringValue: require('es-abstract/2015/thisStringValue'),
	thisTimeValue: require('es-abstract/2015/thisTimeValue'),
	TimeClip: require('es-abstract/2015/TimeClip'),
	TimeFromYear: require('es-abstract/2015/TimeFromYear'),
	TimeWithinDay: require('es-abstract/2015/TimeWithinDay'),
	ToBoolean: require('es-abstract/2015/ToBoolean'),
	ToDateString: require('es-abstract/2015/ToDateString'),
	ToInt16: require('es-abstract/2015/ToInt16'),
	ToInt32: require('es-abstract/2015/ToInt32'),
	ToInt8: require('es-abstract/2015/ToInt8'),
	ToInteger: require('es-abstract/2015/ToInteger'),
	ToLength: require('es-abstract/2015/ToLength'),
	ToNumber: require('es-abstract/2015/ToNumber'),
	ToObject: require('es-abstract/2015/ToObject'),
	ToPrimitive: require('es-abstract/2015/ToPrimitive'),
	ToPropertyDescriptor: require('es-abstract/2015/ToPropertyDescriptor'),
	ToPropertyKey: require('es-abstract/2015/ToPropertyKey'),
	ToString: require('es-abstract/2015/ToString'),
	ToUint16: require('es-abstract/2015/ToUint16'),
	ToUint32: require('es-abstract/2015/ToUint32'),
	ToUint8: require('es-abstract/2015/ToUint8'),
	ToUint8Clamp: require('es-abstract/2015/ToUint8Clamp'),
	Type: require('es-abstract/2015/Type'),
	ValidateAndApplyPropertyDescriptor: require('es-abstract/2015/ValidateAndApplyPropertyDescriptor'),
	WeekDay: require('es-abstract/2015/WeekDay'),
	YearFromTime: require('es-abstract/2015/YearFromTime')
};

module.exports = ES6;
