// mocha/lib/reporters/index.js
'use strict';

// Alias exports to a their normalized format Mocha#reporter to prevent a need
// for dynamic (try/catch) requires, which Browserify doesn't handle.
exports.Base = exports.base = require('mocha/lib/reporters/base');
exports.Dot = exports.dot = require('mocha/lib/reporters/dot');
exports.Doc = exports.doc = require('mocha/lib/reporters/doc');
exports.TAP = exports.tap = require('mocha/lib/reporters/tap');
exports.JSON = exports.json = require('mocha/lib/reporters/json');
exports.HTML = exports.html = require('mocha/lib/reporters/html');
exports.List = exports.list = require('mocha/lib/reporters/list');
exports.Min = exports.min = require('mocha/lib/reporters/min');
exports.Spec = exports.spec = require('mocha/lib/reporters/spec');
exports.Nyan = exports.nyan = require('mocha/lib/reporters/nyan');
exports.XUnit = exports.xunit = require('mocha/lib/reporters/xunit');
exports.Markdown = exports.markdown = require('mocha/lib/reporters/markdown');
exports.Progress = exports.progress = require('mocha/lib/reporters/progress');
exports.Landing = exports.landing = require('mocha/lib/reporters/landing');
exports.JSONStream = exports['json-stream'] = require('mocha/lib/reporters/json-stream');
