// es-abstract/es2016.js
'use strict';

/* eslint global-require: 0 */
// https://www.ecma-international.org/ecma-262/7.0/#sec-abstract-operations
var ES2016 = {
	'Abstract Equality Comparison': require('es-abstract/2016/AbstractEqualityComparison'),
	'Abstract Relational Comparison': require('es-abstract/2016/AbstractRelationalComparison'),
	'Strict Equality Comparison': require('es-abstract/2016/StrictEqualityComparison'),
	AdvanceStringIndex: require('es-abstract/2016/AdvanceStringIndex'),
	ArrayCreate: require('es-abstract/2016/ArrayCreate'),
	ArraySetLength: require('es-abstract/2016/ArraySetLength'),
	ArraySpeciesCreate: require('es-abstract/2016/ArraySpeciesCreate'),
	Call: require('es-abstract/2016/Call'),
	CanonicalNumericIndexString: require('es-abstract/2016/CanonicalNumericIndexString'),
	CompletePropertyDescriptor: require('es-abstract/2016/CompletePropertyDescriptor'),
	CreateDataProperty: require('es-abstract/2016/CreateDataProperty'),
	CreateDataPropertyOrThrow: require('es-abstract/2016/CreateDataPropertyOrThrow'),
	CreateHTML: require('es-abstract/2016/CreateHTML'),
	CreateIterResultObject: require('es-abstract/2016/CreateIterResultObject'),
	CreateListFromArrayLike: require('es-abstract/2016/CreateListFromArrayLike'),
	CreateMethodProperty: require('es-abstract/2016/CreateMethodProperty'),
	DateFromTime: require('es-abstract/2016/DateFromTime'),
	Day: require('es-abstract/2016/Day'),
	DayFromYear: require('es-abstract/2016/DayFromYear'),
	DaysInYear: require('es-abstract/2016/DaysInYear'),
	DayWithinYear: require('es-abstract/2016/DayWithinYear'),
	DefinePropertyOrThrow: require('es-abstract/2016/DefinePropertyOrThrow'),
	DeletePropertyOrThrow: require('es-abstract/2016/DeletePropertyOrThrow'),
	EnumerableOwnNames: require('es-abstract/2016/EnumerableOwnNames'),
	FromPropertyDescriptor: require('es-abstract/2016/FromPropertyDescriptor'),
	Get: require('es-abstract/2016/Get'),
	GetIterator: require('es-abstract/2016/GetIterator'),
	GetMethod: require('es-abstract/2016/GetMethod'),
	GetOwnPropertyKeys: require('es-abstract/2016/GetOwnPropertyKeys'),
	GetPrototypeFromConstructor: require('es-abstract/2016/GetPrototypeFromConstructor'),
	GetSubstitution: require('es-abstract/2016/GetSubstitution'),
	GetV: require('es-abstract/2016/GetV'),
	HasOwnProperty: require('es-abstract/2016/HasOwnProperty'),
	HasProperty: require('es-abstract/2016/HasProperty'),
	HourFromTime: require('es-abstract/2016/HourFromTime'),
	InLeapYear: require('es-abstract/2016/InLeapYear'),
	InstanceofOperator: require('es-abstract/2016/InstanceofOperator'),
	Invoke: require('es-abstract/2016/Invoke'),
	IsAccessorDescriptor: require('es-abstract/2016/IsAccessorDescriptor'),
	IsArray: require('es-abstract/2016/IsArray'),
	IsCallable: require('es-abstract/2016/IsCallable'),
	IsConcatSpreadable: require('es-abstract/2016/IsConcatSpreadable'),
	IsConstructor: require('es-abstract/2016/IsConstructor'),
	IsDataDescriptor: require('es-abstract/2016/IsDataDescriptor'),
	IsExtensible: require('es-abstract/2016/IsExtensible'),
	IsGenericDescriptor: require('es-abstract/2016/IsGenericDescriptor'),
	IsInteger: require('es-abstract/2016/IsInteger'),
	IsPromise: require('es-abstract/2016/IsPromise'),
	IsPropertyDescriptor: require('es-abstract/2016/IsPropertyDescriptor'),
	IsPropertyKey: require('es-abstract/2016/IsPropertyKey'),
	IsRegExp: require('es-abstract/2016/IsRegExp'),
	IterableToArrayLike: require('es-abstract/2016/IterableToArrayLike'),
	IteratorClose: require('es-abstract/2016/IteratorClose'),
	IteratorComplete: require('es-abstract/2016/IteratorComplete'),
	IteratorNext: require('es-abstract/2016/IteratorNext'),
	IteratorStep: require('es-abstract/2016/IteratorStep'),
	IteratorValue: require('es-abstract/2016/IteratorValue'),
	MakeDate: require('es-abstract/2016/MakeDate'),
	MakeDay: require('es-abstract/2016/MakeDay'),
	MakeTime: require('es-abstract/2016/MakeTime'),
	MinFromTime: require('es-abstract/2016/MinFromTime'),
	modulo: require('es-abstract/2016/modulo'),
	MonthFromTime: require('es-abstract/2016/MonthFromTime'),
	msFromTime: require('es-abstract/2016/msFromTime'),
	ObjectCreate: require('es-abstract/2016/ObjectCreate'),
	OrdinaryDefineOwnProperty: require('es-abstract/2016/OrdinaryDefineOwnProperty'),
	OrdinaryGetOwnProperty: require('es-abstract/2016/OrdinaryGetOwnProperty'),
	OrdinaryGetPrototypeOf: require('es-abstract/2016/OrdinaryGetPrototypeOf'),
	OrdinarySetPrototypeOf: require('es-abstract/2016/OrdinarySetPrototypeOf'),
	OrdinaryHasInstance: require('es-abstract/2016/OrdinaryHasInstance'),
	OrdinaryHasProperty: require('es-abstract/2016/OrdinaryHasProperty'),
	RegExpExec: require('es-abstract/2016/RegExpExec'),
	RequireObjectCoercible: require('es-abstract/2016/RequireObjectCoercible'),
	SameValue: require('es-abstract/2016/SameValue'),
	SameValueNonNumber: require('es-abstract/2016/SameValueNonNumber'),
	SameValueZero: require('es-abstract/2016/SameValueZero'),
	SecFromTime: require('es-abstract/2016/SecFromTime'),
	Set: require('es-abstract/2016/Set'),
	SetFunctionName: require('es-abstract/2016/SetFunctionName'),
	SetIntegrityLevel: require('es-abstract/2016/SetIntegrityLevel'),
	SpeciesConstructor: require('es-abstract/2016/SpeciesConstructor'),
	SymbolDescriptiveString: require('es-abstract/2016/SymbolDescriptiveString'),
	TestIntegrityLevel: require('es-abstract/2016/TestIntegrityLevel'),
	thisBooleanValue: require('es-abstract/2016/thisBooleanValue'),
	thisNumberValue: require('es-abstract/2016/thisNumberValue'),
	thisStringValue: require('es-abstract/2016/thisStringValue'),
	thisTimeValue: require('es-abstract/2016/thisTimeValue'),
	TimeClip: require('es-abstract/2016/TimeClip'),
	TimeFromYear: require('es-abstract/2016/TimeFromYear'),
	TimeWithinDay: require('es-abstract/2016/TimeWithinDay'),
	ToBoolean: require('es-abstract/2016/ToBoolean'),
	ToDateString: require('es-abstract/2016/ToDateString'),
	ToInt16: require('es-abstract/2016/ToInt16'),
	ToInt32: require('es-abstract/2016/ToInt32'),
	ToInt8: require('es-abstract/2016/ToInt8'),
	ToInteger: require('es-abstract/2016/ToInteger'),
	ToLength: require('es-abstract/2016/ToLength'),
	ToNumber: require('es-abstract/2016/ToNumber'),
	ToObject: require('es-abstract/2016/ToObject'),
	ToPrimitive: require('es-abstract/2016/ToPrimitive'),
	ToPropertyDescriptor: require('es-abstract/2016/ToPropertyDescriptor'),
	ToPropertyKey: require('es-abstract/2016/ToPropertyKey'),
	ToString: require('es-abstract/2016/ToString'),
	ToUint16: require('es-abstract/2016/ToUint16'),
	ToUint32: require('es-abstract/2016/ToUint32'),
	ToUint8: require('es-abstract/2016/ToUint8'),
	ToUint8Clamp: require('es-abstract/2016/ToUint8Clamp'),
	Type: require('es-abstract/2016/Type'),
	ValidateAndApplyPropertyDescriptor: require('es-abstract/2016/ValidateAndApplyPropertyDescriptor'),
	WeekDay: require('es-abstract/2016/WeekDay'),
	YearFromTime: require('es-abstract/2016/YearFromTime')
};

module.exports = ES2016;
