// es-abstract/es2019.js
'use strict';

/* eslint global-require: 0 */
// https://www.ecma-international.org/ecma-262/10.0/#sec-abstract-operations
var ES2019 = {
	'Abstract Equality Comparison': require('es-abstract/2019/AbstractEqualityComparison'),
	'Abstract Relational Comparison': require('es-abstract/2019/AbstractRelationalComparison'),
	'Strict Equality Comparison': require('es-abstract/2019/StrictEqualityComparison'),
	AddEntriesFromIterable: require('es-abstract/2019/AddEntriesFromIterable'),
	AdvanceStringIndex: require('es-abstract/2019/AdvanceStringIndex'),
	ArrayCreate: require('es-abstract/2019/ArrayCreate'),
	ArraySetLength: require('es-abstract/2019/ArraySetLength'),
	ArraySpeciesCreate: require('es-abstract/2019/ArraySpeciesCreate'),
	Call: require('es-abstract/2019/Call'),
	CanonicalNumericIndexString: require('es-abstract/2019/CanonicalNumericIndexString'),
	CompletePropertyDescriptor: require('es-abstract/2019/CompletePropertyDescriptor'),
	CopyDataProperties: require('es-abstract/2019/CopyDataProperties'),
	CreateDataProperty: require('es-abstract/2019/CreateDataProperty'),
	CreateDataPropertyOrThrow: require('es-abstract/2019/CreateDataPropertyOrThrow'),
	CreateHTML: require('es-abstract/2019/CreateHTML'),
	CreateIterResultObject: require('es-abstract/2019/CreateIterResultObject'),
	CreateListFromArrayLike: require('es-abstract/2019/CreateListFromArrayLike'),
	CreateMethodProperty: require('es-abstract/2019/CreateMethodProperty'),
	DateFromTime: require('es-abstract/2019/DateFromTime'),
	DateString: require('es-abstract/2019/DateString'),
	Day: require('es-abstract/2019/Day'),
	DayFromYear: require('es-abstract/2019/DayFromYear'),
	DaysInYear: require('es-abstract/2019/DaysInYear'),
	DayWithinYear: require('es-abstract/2019/DayWithinYear'),
	DefinePropertyOrThrow: require('es-abstract/2019/DefinePropertyOrThrow'),
	DeletePropertyOrThrow: require('es-abstract/2019/DeletePropertyOrThrow'),
	EnumerableOwnPropertyNames: require('es-abstract/2019/EnumerableOwnPropertyNames'),
	FlattenIntoArray: require('es-abstract/2019/FlattenIntoArray'),
	FromPropertyDescriptor: require('es-abstract/2019/FromPropertyDescriptor'),
	Get: require('es-abstract/2019/Get'),
	GetIterator: require('es-abstract/2019/GetIterator'),
	GetMethod: require('es-abstract/2019/GetMethod'),
	GetOwnPropertyKeys: require('es-abstract/2019/GetOwnPropertyKeys'),
	GetPrototypeFromConstructor: require('es-abstract/2019/GetPrototypeFromConstructor'),
	GetSubstitution: require('es-abstract/2019/GetSubstitution'),
	GetV: require('es-abstract/2019/GetV'),
	HasOwnProperty: require('es-abstract/2019/HasOwnProperty'),
	HasProperty: require('es-abstract/2019/HasProperty'),
	HourFromTime: require('es-abstract/2019/HourFromTime'),
	InLeapYear: require('es-abstract/2019/InLeapYear'),
	InstanceofOperator: require('es-abstract/2019/InstanceofOperator'),
	Invoke: require('es-abstract/2019/Invoke'),
	IsAccessorDescriptor: require('es-abstract/2019/IsAccessorDescriptor'),
	IsArray: require('es-abstract/2019/IsArray'),
	IsCallable: require('es-abstract/2019/IsCallable'),
	IsConcatSpreadable: require('es-abstract/2019/IsConcatSpreadable'),
	IsConstructor: require('es-abstract/2019/IsConstructor'),
	IsDataDescriptor: require('es-abstract/2019/IsDataDescriptor'),
	IsExtensible: require('es-abstract/2019/IsExtensible'),
	IsGenericDescriptor: require('es-abstract/2019/IsGenericDescriptor'),
	IsInteger: require('es-abstract/2019/IsInteger'),
	IsPromise: require('es-abstract/2019/IsPromise'),
	IsPropertyKey: require('es-abstract/2019/IsPropertyKey'),
	IsRegExp: require('es-abstract/2019/IsRegExp'),
	IsStringPrefix: require('es-abstract/2019/IsStringPrefix'),
	IterableToList: require('es-abstract/2019/IterableToList'),
	IteratorClose: require('es-abstract/2019/IteratorClose'),
	IteratorComplete: require('es-abstract/2019/IteratorComplete'),
	IteratorNext: require('es-abstract/2019/IteratorNext'),
	IteratorStep: require('es-abstract/2019/IteratorStep'),
	IteratorValue: require('es-abstract/2019/IteratorValue'),
	MakeDate: require('es-abstract/2019/MakeDate'),
	MakeDay: require('es-abstract/2019/MakeDay'),
	MakeTime: require('es-abstract/2019/MakeTime'),
	MinFromTime: require('es-abstract/2019/MinFromTime'),
	modulo: require('es-abstract/2019/modulo'),
	MonthFromTime: require('es-abstract/2019/MonthFromTime'),
	msFromTime: require('es-abstract/2019/msFromTime'),
	NumberToString: require('es-abstract/2019/NumberToString'),
	ObjectCreate: require('es-abstract/2019/ObjectCreate'),
	OrdinaryDefineOwnProperty: require('es-abstract/2019/OrdinaryDefineOwnProperty'),
	OrdinaryGetOwnProperty: require('es-abstract/2019/OrdinaryGetOwnProperty'),
	OrdinaryGetPrototypeOf: require('es-abstract/2019/OrdinaryGetPrototypeOf'),
	OrdinarySetPrototypeOf: require('es-abstract/2019/OrdinarySetPrototypeOf'),
	OrdinaryHasInstance: require('es-abstract/2019/OrdinaryHasInstance'),
	OrdinaryHasProperty: require('es-abstract/2019/OrdinaryHasProperty'),
	PromiseResolve: require('es-abstract/2019/PromiseResolve'),
	RegExpExec: require('es-abstract/2019/RegExpExec'),
	RequireObjectCoercible: require('es-abstract/2019/RequireObjectCoercible'),
	SameValue: require('es-abstract/2019/SameValue'),
	SameValueNonNumber: require('es-abstract/2019/SameValueNonNumber'),
	SameValueZero: require('es-abstract/2019/SameValueZero'),
	SecFromTime: require('es-abstract/2019/SecFromTime'),
	Set: require('es-abstract/2019/Set'),
	SetFunctionName: require('es-abstract/2019/SetFunctionName'),
	SetIntegrityLevel: require('es-abstract/2019/SetIntegrityLevel'),
	SpeciesConstructor: require('es-abstract/2019/SpeciesConstructor'),
	SymbolDescriptiveString: require('es-abstract/2019/SymbolDescriptiveString'),
	TestIntegrityLevel: require('es-abstract/2019/TestIntegrityLevel'),
	thisBooleanValue: require('es-abstract/2019/thisBooleanValue'),
	thisNumberValue: require('es-abstract/2019/thisNumberValue'),
	thisStringValue: require('es-abstract/2019/thisStringValue'),
	thisSymbolValue: require('es-abstract/2019/thisSymbolValue'),
	thisTimeValue: require('es-abstract/2019/thisTimeValue'),
	TimeClip: require('es-abstract/2019/TimeClip'),
	TimeFromYear: require('es-abstract/2019/TimeFromYear'),
	TimeString: require('es-abstract/2019/TimeString'),
	TimeWithinDay: require('es-abstract/2019/TimeWithinDay'),
	ToBoolean: require('es-abstract/2019/ToBoolean'),
	ToDateString: require('es-abstract/2019/ToDateString'),
	ToIndex: require('es-abstract/2019/ToIndex'),
	ToInt16: require('es-abstract/2019/ToInt16'),
	ToInt32: require('es-abstract/2019/ToInt32'),
	ToInt8: require('es-abstract/2019/ToInt8'),
	ToInteger: require('es-abstract/2019/ToInteger'),
	ToLength: require('es-abstract/2019/ToLength'),
	ToNumber: require('es-abstract/2019/ToNumber'),
	ToObject: require('es-abstract/2019/ToObject'),
	ToPrimitive: require('es-abstract/2019/ToPrimitive'),
	ToPropertyDescriptor: require('es-abstract/2019/ToPropertyDescriptor'),
	ToPropertyKey: require('es-abstract/2019/ToPropertyKey'),
	ToString: require('es-abstract/2019/ToString'),
	ToUint16: require('es-abstract/2019/ToUint16'),
	ToUint32: require('es-abstract/2019/ToUint32'),
	ToUint8: require('es-abstract/2019/ToUint8'),
	ToUint8Clamp: require('es-abstract/2019/ToUint8Clamp'),
	TrimString: require('es-abstract/2019/TrimString'),
	Type: require('es-abstract/2019/Type'),
	ValidateAndApplyPropertyDescriptor: require('es-abstract/2019/ValidateAndApplyPropertyDescriptor'),
	WeekDay: require('es-abstract/2019/WeekDay'),
	YearFromTime: require('es-abstract/2019/YearFromTime')
};

module.exports = ES2019;
