// es-abstract/es5.js
'use strict';

/* eslint global-require: 0 */

// https://es5.github.io/#x9
module.exports = {
	'Abstract Equality Comparison': require('es-abstract/5/AbstractEqualityComparison'),
	'Abstract Relational Comparison': require('es-abstract/5/AbstractRelationalComparison'),
	'Strict Equality Comparison': require('es-abstract/5/StrictEqualityComparison'),
	CheckObjectCoercible: require('es-abstract/5/CheckObjectCoercible'),
	DateFromTime: require('es-abstract/5/DateFromTime'),
	Day: require('es-abstract/5/Day'),
	DayFromYear: require('es-abstract/5/DayFromYear'),
	DaysInYear: require('es-abstract/5/DaysInYear'),
	DayWithinYear: require('es-abstract/5/DayWithinYear'),
	FromPropertyDescriptor: require('es-abstract/5/FromPropertyDescriptor'),
	HourFromTime: require('es-abstract/5/HourFromTime'),
	InLeapYear: require('es-abstract/5/InLeapYear'),
	IsAccessorDescriptor: require('es-abstract/5/IsAccessorDescriptor'),
	IsCallable: require('es-abstract/5/IsCallable'),
	IsDataDescriptor: require('es-abstract/5/IsDataDescriptor'),
	IsGenericDescriptor: require('es-abstract/5/IsGenericDescriptor'),
	IsPropertyDescriptor: require('es-abstract/5/IsPropertyDescriptor'),
	MakeDate: require('es-abstract/5/MakeDate'),
	MakeDay: require('es-abstract/5/MakeDay'),
	MakeTime: require('es-abstract/5/MakeTime'),
	MinFromTime: require('es-abstract/5/MinFromTime'),
	modulo: require('es-abstract/5/modulo'),
	MonthFromTime: require('es-abstract/5/MonthFromTime'),
	msFromTime: require('es-abstract/5/msFromTime'),
	SameValue: require('es-abstract/5/SameValue'),
	SecFromTime: require('es-abstract/5/SecFromTime'),
	TimeClip: require('es-abstract/5/TimeClip'),
	TimeFromYear: require('es-abstract/5/TimeFromYear'),
	TimeWithinDay: require('es-abstract/5/TimeWithinDay'),
	ToBoolean: require('es-abstract/5/ToBoolean'),
	ToInt32: require('es-abstract/5/ToInt32'),
	ToInteger: require('es-abstract/5/ToInteger'),
	ToNumber: require('es-abstract/5/ToNumber'),
	ToObject: require('es-abstract/5/ToObject'),
	ToPrimitive: require('es-abstract/5/ToPrimitive'),
	ToPropertyDescriptor: require('es-abstract/5/ToPropertyDescriptor'),
	ToString: require('es-abstract/5/ToString'),
	ToUint16: require('es-abstract/5/ToUint16'),
	ToUint32: require('es-abstract/5/ToUint32'),
	Type: require('es-abstract/5/Type'),
	WeekDay: require('es-abstract/5/WeekDay'),
	YearFromTime: require('es-abstract/5/YearFromTime')
};
